experimenter/experimenter/nimbus-ui/src/components/Summary/index.tsx

```diff
@@ -38,6 +44,51 @@ const Summary = ({ experiment, refetch }: SummaryProps) => {
   const takeawaysProps = useTakeaways(experiment, refetch);
   const qaProps = useQA(experiment, refetch);
   const status = getStatus(experiment);
+  const [legalSignoff, setLegalSignoff] = useState(
+    experiment.legalSignoff || false,
+  );
+  const [qaSignoff, setQaSignoff] = useState(experiment.qaSignoff || false);
+  const [vpSignoff, setVpSignoff] = useState(experiment.vpSignoff || false);
+  const [updateExperiment] = useMutation<
+    updateExperiment,
+    updateExperimentVariables
+  >(UPDATE_EXPERIMENT_MUTATION);
+
+  const handleLegalSignoffChange = (value: boolean) => {
+    setLegalSignoff(value);
+    updateExperimentSignoff(value, qaSignoff, vpSignoff);
+  };
+
+  const handleQaSignoffChange = (value: boolean) => {
+    setQaSignoff(value);
+    updateExperimentSignoff(legalSignoff, value, vpSignoff);
+  };
+
+  const handleVpSignoffChange = (value: boolean) => {
+    setVpSignoff(value);
+    updateExperimentSignoff(legalSignoff, qaSignoff, value);
+  };
+  const updateExperimentSignoff = async (
+    legal: boolean,
+    qa: boolean,
+    vp: boolean,
+  ) => {
+    try {
+      const result = await updateExperiment({
+        variables: {
+          input: {
+            id: experiment.id,
+            legalSignoff: legal,
+            qaSignoff: qa,
+            vpSignoff: vp,
+            changelogMessage: CHANGELOG_MESSAGES.UPDATE_SIGNOFF,
+          },
+        },
+      });
+    } catch (error) {
+      console.error("Error updating sign-off values:", error);
```

Comment:
Does this mean we wont propagate errors to sentry?

Reply:
I believe a sentry wrapper should exist and report when we do console.error